lib/ansible/module_utils/facts.py

```diff
@@ -937,6 +937,18 @@ def get_dmi_facts(self):
     @timeout(10)
     def get_mount_facts(self):
         self.facts['mounts'] = []
+        bind_mounts = []
+        findmntPath = module.get_bin_path("findmnt")
+        rc, out, err = module.run_command("%s -lnur" % ( findmntPath ), use_unsafe_shell=True)
```

Comment:
I tested against Debian squeeze systems (which don't have findmnt) and it worked, because run_command returns rc != 0 anyway on non-existing commands.

But I guess it's cleaner to not rely on that behavior, I'll update the code (but not today).